Trading/Mode/dca_trading_mode/dca_trading.py

```diff
@@ -0,0 +1,144 @@
+
+        except (trading_errors.MissingFunds, trading_errors.MissingMinimalExchangeTradeVolume):
+            raise
+        except Exception as e:
+            self.logger.error(f"Failed to create order : {e}. Order: "
+                              f"{current_order.get_string_info() if current_order else None}")
+            self.logger.exception(e, False)
+            return []
+
+    async def can_create_order(self, symbol, state):
+        can_create_order_result = await super().can_create_order(symbol, state)
+        if not can_create_order_result:
+            currency, market = symbol_util.split_symbol(symbol)
+            self.logger.error(f"Can't create order : not enough balance. Please deposit more {market}.")
+        return can_create_order_result
+
+
+class DCATradingModeProducer(trading_modes.AbstractTradingModeProducer):
+    MINUTES_BEFORE_NEXT_BUY = "minutes_before_next_buy"
+    
+    def __init__(self, channel, config, trading_mode, exchange_manager):
+        super().__init__(channel, config, trading_mode, exchange_manager)
+        self.task = None
+        self.state = trading_enums.EvaluatorStates.LONG
+        self.minutes_before_next_buy = self.trading_mode.trading_config.get(self.MINUTES_BEFORE_NEXT_BUY, 60)
+
+    async def stop(self):
+        if self.trading_mode is not None:
+            self.trading_mode.consumers[0].flush()
+        await super().stop()
+
+    async def trigger_dca_for_symbol(self, cryptocurrency, symbol):
+        # call orders creation from consumers
+        await self.submit_trading_evaluation(cryptocurrency=cryptocurrency,
+                                             symbol=symbol,
+                                             time_frame=None,
+                                             final_note=None,
+                                             state=self.state)
+
+        # send_notification
+        if not self.exchange_manager.is_backtesting:
+            await self._send_alert_notification(None)
+
+    async def dca_task(self):
+        while not self.should_stop:
+            try:
+                await asyncio.sleep(self.minutes_before_next_buy)
+                self.logger.info("DCA task triggered")
+
+                for cryptocurrency, pairs in trading_util.get_traded_pairs_by_currency(self.exchange_manager.config).items():
+                    for pair in pairs:
+                        await self.trigger_dca_for_symbol(cryptocurrency=cryptocurrency, symbol=pair)
+                
+            except Exception as e:
+                self.logger.error(f"An error happened during DCA task : {e}")
+
+    async def start(self) -> None:
+        self.task = await asyncio.create_task(self.dca_task())
+
+    async def _send_alert_notification(self, symbol):
+        try:
+            import octobot_services.api as services_api
+            import octobot_services.enums as services_enum
+            title = f"OCTOBOT DCA ALERT : #{symbol}"
```

Comment:
I think we should use something else than "ALERT" here: "alert" can be scarry to users while this is an informative message, maybe "something like":
title = f"DCA trigger for #{symbol}"
(I also don't think we need "OctoBot", notifications are always in an OctoBot context already)